src/main/java/com/softserve/academy/spaced/repetition/service/CardService.java

```diff
@@ -32,11 +32,13 @@ public Card getCard(Long id) {
 
     public void addCard(Card card, Long deckId) {
         Deck deck = deckRepository.findOne(deckId);
+        card.setDeck(deck);
         deck.getCards().add(cardRepository.save(card));
     }
 
     public void updateCard(Long id, Card card) {
         card.setId(id);
```

Comment:
This method does nothing?
As for me, it reads from DB and saves the same into.

Reply:
I don't think so. It sets the id of the deck to the card, that is it adds that card to the deck.

Reply:
Oh, I'm sorry. I thought of the previous method earlier) You seem to be right.

Reply:
Oh, I'm sorry. I thought of the previous method earlier) You seem to be right.

Reply:
Is the problem only in the name?